test/distributed/cases/array/array.sql

```diff
@@ -0,0 +1,80 @@
+-- pre
+drop database if exists vecdb;
+create database vecdb;
```

Comment:
add some cases like "SELECT * into outfile 'a.txt'" using the vector type

Reply:
https://github.com/matrixorigin/matrixone/issues/11446